app/helpers/work_packages_helper.rb

```diff
@@ -327,22 +327,33 @@ def work_package_show_attributes(work_package)
     group_work_package_attributes work_package_show_attribute_list(work_package)
   end
 
-  def work_package_show_table_row(attribute, klass = nil, attribute_lang = nil, value_lang = nil, &block)
-    klass = attribute.to_s.dasherize if klass.nil?
+  def work_package_show_dd_dt(attribute, css_class = nil, attribute_lang = nil, value_lang = nil, &block)
+    css_classes = if css_class.nil?
+      [ "-#{attribute.to_s.dasherize}"]
```

Comment:
Use 2 (not -12) spaces for indentation.<br>Space inside square brackets detected.